presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -1108,6 +1117,13 @@ public void testFilterFunctionsWithOptimization()
                 "SELECT partkey FROM lineitem WHERE orderkey > 10");
     }
 
+    @Test
+    public void testOptimizeWithWarningMessage()
+    {
+        assertWarnings(getQueryRunner(), getSession(), "SELECT * FROM lineitem WHERE orderkey = 1 and orderkey = 2",
+                ImmutableSet.of(HIVE_TABLESCAN_CONVERTED_TO_VALUESNODE.toWarningCode()));
```

Comment:
nit: one param per inline with the first line empty:
```java
assertWarnings(
        getQueryRunner(),
        getSession(),
        "SELECT * FROM lineitem WHERE orderkey = 1 and orderkey = 2",
        ImmutableSet.of(HIVE_TABLESCAN_CONVERTED_TO_VALUESNODE.toWarningCode()));

Reply:
Done